File: leetcode_java/1832.CheckIfTheSentenceIsPangram.java
class Solution {
    public boolean checkIfPangram(String sentence) {
        HashMap<Character, Integer> map = new HashMap<>();

        int count = 0;
        for (int i = 0; i < sentence.length(); ++ i) {
            char ch = sentence.charAt(i);
            map.put(ch, 1 + map.getOrDefault(ch, 0));
            if (map.get(ch) == 1) ++ count;
        }
        return count == 26;
    }
}

/**
1832. Check if the Sentence Is Pangram

A pangram is a sentence where every letter of the English alphabet appears at least once.

Given a string sentence containing only lowercase English letters, return true if sentence 
is a pangram, or false otherwise.
 */

File: leetcode_python/1832. Check if the Sentence Is Pangram.py
class Solution:
    def checkIfPangram(self, sentence: str) -> bool:
        charCount = [0] * 26

        for ch in sentence:
            charCount[ord(ch) - ord('a')] += 1
        
        for ct in charCount:
            if ct == 0:
                return False
        return True

/**
1832. Check if the Sentence Is Pangram
Solved
Easy
Topics
conpanies iconCompanies
Hint

A pangram is a sentence where every letter of the English alphabet appears at least once.

Given a string sentence containing only lowercase English letters, return true if sentence is a pangram, or false otherwise.

 

Example 1:

Input: sentence = "thequickbrownfoxjumpsoverthelazydog"
Output: true
Explanation: sentence contains at least one of every letter of the English alphabet.

Example 2:

Input: sentence = "leetcode"
Output: false
 */
